src/components/common/ColonyCreationWizard/StepConfirmAllInput.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+
+import { WizardStepProps } from '~shared/Wizard';
+import Heading from '~shared/Heading';
+import Button from '~shared/Button';
+import { ActionForm, FormStatus } from '~shared/Fields';
```

Comment:
Was Formik updated to the latest version ? Can you please check ?

Reply:
@rdig Yep, latest is 2.2.9 which is our version in the Cdapp. 

As an aside, Formik is no longer being maintained (or at least, that's the impression I get from its last update being a year old, and there being over 600 outstanding issues in the repo. Plus Jared Palmer has moved on to turbopack, which suggests he's got other things to care about). 

I've noticed a few things that make formik a little annoying to work with, the most notable of which being that it doesn't reliably validate the latest form state. There are a number of issues in the formik repo that refer to this, some of which have been addressed, some are still open. [He essentially says it's a problem with hooks and can't be fixed](https://github.com/jaredpalmer/formik/issues/529#issuecomment-571294217). I've calling `validate` inside a `setTimeout` as a workaround, which does the job, but is a bit ugly. 

I understand that it's not currently high priority, but may be worth considering migrating to something else at some point like [react hook form](https://react-hook-form.com/), which has 30+k stars on github and is currently maintained (0 outstanding issues). Or using it to build new forms, and considering formik as legacy.

Reply:
I'm happy to migrate, but we need to have a call an figure out what the exact scope of this undertaking will be.

Also, I really appreciate you took time to research this. :+1:

Reply:
Awesome thanks. I have sent an invite for Thursday.